e2e/tests/core/client_test.go

```diff
@@ -67,7 +67,7 @@ func (s *ClientTestSuite) TestClientUpdateProposal_Succeeds() {
 		relayer            ibc.Relayer
 		subjectClientID    string
 		substituteClientID string
-		badTrustingPeriod  = time.Duration(time.Second)
+		badTrustingPeriod  = time.Duration(time.Second * 10)
```

Comment:
this is what caused the e2e test to fail. apparently with a one second set for badTrustingPeriod, the client was immediately seen as expired (and the new check added caused it to fail immediately). Bumping it a bit seems to do the trick just fine.

Reply:
wow! That's actually kind of nice. Should we add a comment about the 10 seconds:
```go
// set the trusting period to a value which will still be valid upon client creation, but invalid before the first update
```